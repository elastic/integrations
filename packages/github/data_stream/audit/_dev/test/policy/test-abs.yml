input: azure-blob-storage
vars:
data_stream:
  vars:
    account_name: devstoreaccount1
    service_account_key: "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
    number_of_workers: 3
    poll: true
    poll_interval: 15s
    containers: |
      - name: test-container
    preserve_original_event: true
    preserve_duplicate_custom_fields: true
