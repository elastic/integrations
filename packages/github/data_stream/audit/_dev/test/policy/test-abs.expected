inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: github
      name: test-abs-github
      streams:
        - account_name: devstoreaccount1
          auth.shared_credentials.account_key: ${SECRET_0}
          containers:
            - name: test-container
          data_stream:
            dataset: github.audit
            type: logs
          file_selectors: null
          max_workers: 3
          poll: true
          poll_interval: 15s
          publisher_pipeline.disable_host: true
          tags:
            - preserve_original_event
            - preserve_duplicate_custom_fields
            - forwarded
            - github.audit
      type: azure-blob-storage
      use_output: default
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - logs-github.audit-ep
                  privileges:
                    - auto_configure
                    - create_doc
secret_references:
    - {}
